docs/dev/annual_updates.rst

```diff
@@ -156,57 +142,140 @@ B. FERC Form 1
+CSV files stored under ``src/pudl/package_data/ferc1/row_maps`` and organized it by
+original FERC 1 DB table name.
+
+The FERC 1 data from 2021 onwards (XBRL files) is organized into a taxonomy that keeps
+it relatively consistent year to year. We keep track of the connection between DBF and
+XBRL tables in the ``src/pudl/package_data/ferc1/dbf_to_xbrl_tables.csv`` and
+``src/pudl/package_data/ferc1/dbf_to_xbrl.csv`` files.
+
+**4.B.1)** If there are any new tables or reason to believe that the xbrl taxonomy has
+changed, revisit the ``dbf_to_xbrl_tables.csv`` and ``dbf_to_xbrl.csv`` files and map
+the tables and records to one another.
+
+.. note::
 
-**5.2)** Differentiate between columns which should be harvested from the transformed
+    **How to use the mapping spreadsheets:**
+
+    The ``dbf_to_xbrl.csv`` maps row numbers from the DBF data with taxonomy factoids
+    from the XBRL data therefore allowing us to merge the data into one continuous
+    timeseries. The ``row_literal`` column is the DBF label for the ``row_number`` in
+    question. This ``row_literal`` must be mapped to an ``xbrl_factoid`` from the XBRL
+    data. These ``xbrl_factoid`` entires are the value columns from the raw XBRL data.
+
+    Look at the ``row_literal`` values for a given table and see which XBRL columns they
+    coorespond to. It's helpful to
+    `view the XBRL taxonomy <https://xbrlview.ferc.gov/>`__ for the table in question.
+
+    The ``row_literals`` may contain elements of the FERC 1 form such as
+    headers that don't map to an XBRL column. These can be marked as ``headers`` in the
+    ``row_type`` column. Other values are either marked as ``report_value`` or
+    ``calculated_value`` to indicate whether the XBRL data is original or can be
+    calculated from other fields. The way to check this is by looking at the online
+    taxonomy and clicking on the "Relationships" tab for a given field. If there are
+    relationships it's probably a ``calculated_value`` and if there are not, it's a
+    ``reported_value``
+
+    The ``dbf_only`` column is marked ``TRUE`` if the ``row_literal`` only shows up in
+    the DBF files. An common example is when several fields are aggregated in the DBF
+    data but not in XBRL. The ``notes`` column is a place to indicate complexity or
+    reasoning and is intended for humans (vs. computers) to read.
+
+
+**4.B.2)** Use the FERC 1 debugging notebook ``devtools/ferc1-etl-debug.ipynb`` to run
+the transforms for each table. Heed any errors or warnings that pop up in the
+logs. One of the most likely bugs will be uncategorized strings. Any column listed in
+:py:const:`pudl.transform.params.ferc1.TRANSFORM_PARAMS` under ``categorize_strings``
+gets cleaned according to varables defined in :mod:`pudl.transform.params.ferc1`.
+Uncategorized string values (think new, strange fuel type spellings) will throw an error
+and should be added to the appropriate dictionaries in
+:mod:`pudl.transform.params.ferc1`.
```

Comment:
I'd personally rather this context live in the warning itself. I can imagine us forgetting to update this if we change it bc it is not close to the code.

Reply:
Ok I removed the specifics here.